src/System.Xml.ReaderWriter/src/System/Xml/AsyncHelper.cs

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace System.Xml
+{
+    internal static class AsyncHelper
+    {
+        public static readonly Task DoneTask = Task.FromResult(true);
```

Comment:
So I should stop reviewing?  What's the goal of submitting this as a PR?  What are you looking for in terms of feedback / sign-off?